web/src/components/admin/settings/AdminQueueTab.vue

```diff
@@ -65,12 +65,9 @@
           <span class="ml-2">{{ task.id }}</span>
           <span class="flex ml-auto gap-2">
             <Badge v-if="task.agent_id !== 0" :label="$t('admin.settings.queue.agent')" :value="task.agent_id" />
-            <Badge
-              v-for="(value, label) in task.labels"
-              :key="label"
-              :label="label.toString()"
-              :value="value || '???'"
-            />
+            <span v-for="(value, label) in task.labels">
+              <Badge v-if="value" :key="label" :label="label.toString()" :value="value" />
+            </span>
```

Comment:
```suggestion
            <template v-for="(value, label) in task.labels">
              <Badge v-if="value" :key="label" :label="label.toString()" :value="value" />
            </template>
```

Would a `template` be enough here as well 🤔